pkg/registry/core/service/storage/rest.go

```diff
@@ -326,7 +326,7 @@ func (rs *REST) healthCheckNodePortUpdate(oldService, service *api.Service, node
 	return true, nil
 }
 
-func (rs *REST) Update(ctx context.Context, name string, objInfo rest.UpdatedObjectInfo, createValidation rest.ValidateObjectFunc, updateValidation rest.ValidateObjectUpdateFunc) (runtime.Object, bool, error) {
+func (rs *REST) Update(ctx context.Context, name string, objInfo rest.UpdatedObjectInfo, createValidation rest.ValidateObjectFunc, updateValidation rest.ValidateObjectUpdateFunc, forceAllowCreate bool) (runtime.Object, bool, error) {
 	oldObj, err := rs.services.Get(ctx, name, &metav1.GetOptions{})
```

Comment:
@jennybuckley just noticed this... this will return a NotFound error and short-circuit, right? need a follow up for this and any other custom Update() impls to honor forceAllowCreate and add tests for it

Reply:
Great catch! I'll make an issue to track it since it's too late for a TODO.
it shouldn't cause any bugs right now because forceAllowCreate will never be set to true yet.

Reply:
sounds good, thanks